client.go

```diff
@@ -865,3 +866,10 @@ type File struct {
 func (f *File) String() string {
 	return fmt.Sprintf("ParamName: %v; FileName: %v", f.ParamName, f.Name)
 }
+
+// MultipartCustomData represent custom data part for multipart request
+type MultipartCustomData struct {
```

Comment:
@Asker80 Change `struct` name to `multipartField`. Basically making as unexported struct.